src/API/Google/AdsAssetGroupAsset.php

```diff
@@ -121,4 +133,84 @@ public function get_assets_by_asset_group_ids( array $asset_groups_ids ): array
 		}
 
 	}
+
+	/**
+	 * Edit assets group assets.
+	 *
+	 * @param int   $asset_group_id The asset group id.
+	 * @param array $assets The assets to create.
+	 *
+	 * @return array The asset group asset operations.
+	 */
+	public function edit_operations( int $asset_group_id, array $assets ): array {
+		if ( empty( $assets ) ) {
+			return [];
+		}
+
+		$assets_operations                    = [];
+		$asset_group_assets_operations        = [];
+		$delete_asset_group_assets_operations = [];
+
+		foreach ( $assets as $asset ) {
+
+			// If content exists, create asset and asset group asset.
+			if ( ! empty( $asset['content'] ) ) {
+				$assets_operations[]             = $this->asset->create_operation( $asset, self::$temporary_id );
+				$asset_group_assets_operations[] = $this->create_operation( $asset_group_id, $asset['field_type'], self::$temporary_id-- );
+			}
+
+			// As Assets are inmmutable, we delete the old link between the asset and the asset group if exists.
+			if ( ! empty( $asset['id'] ) ) {
+				$delete_asset_group_assets_operations[] = $this->delete_operation( $asset_group_id, $asset['field_type'], $asset['id'] );
+			}
+		}
+
+		// Delete asset group assets operations must be executed last so we are never under the minimum quantity.
+		$operations = array_merge( $assets_operations, $asset_group_assets_operations, $delete_asset_group_assets_operations );
+
+		return $operations;
```

Comment:
💅  We can just return `array_merge( $assets_operations, $asset_group_assets_operations, $delete_asset_group_assets_operations )`